journalism_workflow/static/journalism_workflow/photographer/partials/photographer.html

```diff
@@ -0,0 +1,80 @@
+<section class="section-panel">
+  <div class="container-fluid">
+    <div class="row section-header">
+      <div class="col-lg-12 col-md-12 col-sm-12">
+        <h3>
+	  Take and upload photos for the article
+        </h3>
+      </div>
+    </div>
+
+    <!-- Titles for the two columns -->
+    <div class="row padded">
+      <div class="col-lg4 col-md-4 col-sm-4">
+	<h3>Editor's Story Plan</h3>
+      </div>
+
+      <div class="col-lg-8 col-md-8 col-sm-8">
+	<h3>Photographer Instructions</h3>
+      </div>
+
+    </div>
+
+    <div class="row padded">
+
+      <!-- Form with Editor's story plan -->
```

Comment:
Fair, I just meant that if we were going to change how we write directives soon, then I didn't want to write one in the old style now.

I created the 'improve journalism' issue and added a couple wishlist items to it.